### PR TITLE
Huge sysparams refactor

### PR DESCRIPTION
**Pull request by Altreus**

### CODE REVIEW

File: lib/OpusVL/FB11X/SysParams/Controller/SysParams.pm [hunk 1]
```diff
@@ -21,160 +21,66 @@ __PACKAGE__->config
...
+    my $form = $self->edit_form;
     $c->stash->{form} = $form;
-    $c->stash->{param} = $param;
 
-    my $ok = try {
-        $form->process( 
-            params => $c->req->params,
-            init_object => { data_type => 'text' }
-        )
+    $form->process(
```
Comment (Nick-OpusVL) [inline lib/OpusVL/FB11X/SysParams/Controller/SysParams.pm]: Does $form->process by any chance lean on it being DBIC?

File: lib/OpusVL/SysParams/Hat/sysparams/legacy.pm [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package OpusVL::SysParams::Hat::sysparams::legacy;
```
Comment (Nick-OpusVL) [inline lib/OpusVL/SysParams/Hat/sysparams/legacy.pm]: I didn't think we were doing this.

File: lib/OpusVL/SysParams/Hat/sysparams/namespaced.pm [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            __brain => $_[0]->__brain
+        })
+    });
+}
+
+1;
+
+=head1 DESCRIPTION
+
+Wear this hat to provide legacy sysparams.
```
Comment (Nick-OpusVL) [inline lib/OpusVL/SysParams/Hat/sysparams/namespaced.pm]: Use the namespaced hat to provide legacy sysparams?

File: lib/OpusVL/SysParams/Hat/sysparams/legacy.pm [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+our $VERSION = '0';
+# ABSTRACT: Implements sysparams with no regard for namespacing
+
+use Moose;
+use OpusVL::SysParams::Brain::Strategy::Global;
```
Comment (Nick-OpusVL) [inline lib/OpusVL/SysParams/Hat/sysparams/legacy.pm]: Where is the implementation of Strategy::Global?

File: lib/OpusVL/FB11X/SysParams/Controller/SysParams.pm (hunk 1 above)
Comment (Altreus) [reply lib/OpusVL/FB11X/SysParams/Controller/SysParams.pm]: This part of the controller is now a 404 until I can get around to implementing it properly.

**Pull request merged**